src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,53 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import RocketService from '../../services/RocketService';
+
+const initialState = {
+  rockets: [],
+};
+
+const fetchRockets = createAsyncThunk('rockets/fetchRockets', async () => {
+  const response = await RocketService.getAll();
+  const rockets = response.data.map((item) => (
+    {
+      rocket_id: item.id,
+      rocket_name: item.rocket_name,
+      rocket_description: item.description,
+      rocket_image: item.flickr_images[0],
+      rocket_reserved: item.active,
```

Comment:
- [Optional] When fetching rockets, I think no rocket is supposed to me reserved by default.